pyuia/selenium/pageobject.py

```diff
@@ -71,19 +71,30 @@ def find_by(how=None, using=None, multiple=False, cacheable=True, if_exists=Fals
 
     """
     # 'how' AND 'using' take precedence over keyword arguments
-    _how, _using = how, using
-    if not (_how and _using):
+    #_how, _using = how, using
+
+    method_dic = {'_how':how, '_using': using}
+
+    if not (method_dic['_how'] and method_dic['_using']):
         if len(kwargs) != 1 or kwargs.keys()[0] not in _strategy_kwargs.keys() :
             raise ValueError(
                 "If 'how' AND 'using' are not specified, one and only one of the following "
                 "valid keyword arguments should be provided: %s." % _strategy_kwargs.keys())
 
         key = kwargs.keys()[0]
-        _how, _using = _strategy_kwargs[key], kwargs[key]
+        method_dic['_how'], method_dic['_using'] = _strategy_kwargs[key], kwargs[key]
 
     def func(page_object):
         driver = getattr(page_object, driver_attr)
 
+        # For appium v1.5.3, since it doesn't support find by name strategy, we have to adjust our pyuia
+        # We use xpath to replace name
+        # https://github.com/appium/appium/releases/tag/v1.5.0
```

Comment:
The release note only makes an announcement of "Remove long-deprecated name locator strategy" and doesn't suggest alternatives (or workarounds). How about adding some links to documents or discussing threads you referred to for further reference?